_notices/rsn0035.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 35 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation Announcement Dropping CUDA 11.2 support in our Docker Images in Release v23.12"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.12"
+notice_created: 2023-10-05
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-10-05
+---
+
+## Overview
+
+RAPIDS is planning to drop CUDA 11.2 support in our Docker Images in Release `v23.12` release and will remove in its `v24.02` release. `v23.12` will be the last version of RAPIDS to support CUDA 11.12 Docker Images. 
```

Comment:
```suggestion
RAPIDS is planning to deprecate support for its CUDA 11.2 Docker containers in Release `v23.12`, and to cease publishing CUDA 11.2 Docker containers in the `v24.02` release. `v23.12` will be the last version of RAPIDS to distribute CUDA 11.12 Docker containers. 
```

Reply:
Crucial: I see the typo "11.12" instead of "11.2" several places. I fixed this in the PR title and description, and also edited @aravenel's suggestion above.